src/sentry/static/sentry/app/views/projectDashboard/eventNode.jsx

```diff
@@ -25,10 +25,7 @@ const EventNode = createReactClass({
       <li className="group row">
         <div className="col-xs-8 event-details">
           <EventOrGroupHeader data={group} hideIcons hideLevel />
-          <EventOrGroupExtraDetails
-            lastSeen={group.lastSeen}
-            firstSeen={group.firstSeen}
-          />
+          <EventOrGroupExtraDetails {...group} />
```

Comment:
This also fixes:
![image](https://user-images.githubusercontent.com/79684/51406659-a304be80-1b0e-11e9-9a99-58eb852748ba.png)

Reply:
vs 
![image](https://user-images.githubusercontent.com/79684/51406697-b4e66180-1b0e-11e9-87ba-6f58e0a12a53.png)